### DISCUSSION TITLE
Inquiry About GSoC Project

### DISCUSSION THREAD

**Discussion by imperixxl** [openvinotoolkit/openvino#23111] [Google Summer of Code]
Hello everyone,
I'm interested in participating in the OpenVINO Toolkit's GSoC program. Currently, I'm a freshman pursuing my BTech degree. I have limited knowledge about machine learning, but I'm eager to learn through the project.
I've chosen the [OpenVINO adapters - PyTorch adapter](https://github.com/openvinotoolkit/openvino/wiki/Google-Summer-Of-Code#7-openvino-adapters---pytorch-adapter) project because of my interest in implementing PyTorch. I'm currently enrolled in an ML course, which will be followed by one in DL using PyTorch.
I've always been fascinated by open-sourc contributions and would like to start with a "good first issue" on openvino's github. However, if my experience falls short of expectations, I understand.
Thank you for your assistance

**Comment by adrianboguszewski** [comment]
Hey @imperixxl. This project is mentored by @jiwaszki and @akuporos

**Comment by imperixxl** [comment]
Thank you @adrianboguszewski

**Comment by jiwaszki** [comment]
Hi @imperixxl ! I am looking forward to your contribution and our collaboration.

Later this week I will share with you a GFI task that is mandatory prerequisite according to our guidelines: https://github.com/openvinotoolkit/openvino/wiki/Google-Summer-Of-Code#prerequisite-task

I am going to assign you automatically to it. Is it okay with you?

**Reply by imperixxl** [reply]
Hi @jiwaszki Thank you for your message! I'm excited about the opportunity to contribute and collaborate with you.
Regarding the GFI task, I'm more than willing to complete it. However, I'd like to mention that I have less experience in the area. Nevertheless, I'm eager to learn and meet the challenge head-on. Looking forward to working together!

**Reply by jiwaszki** [reply]
@imperixxl here is your task: https://github.com/openvinotoolkit/openvino/issues/21968 -- for some reason GH is not letting me to assign you. Please write `.take` in the task or contact with @p-wysocki on [our Discord channel](https://discord.com/channels/579789537866154054/1186935324279058442) if any other issues appears.

**Comment by imperixxl** [comment]
Thank you @jiwaszki , I will take the task.
